readme.md

```diff
@@ -72,6 +72,15 @@ var value = currency("123.45");
 currency(value);    // 123.45
 ```
 
+Currency is in dollars by default but can be parsed as cents. This will respect the precision option when parsing.
+
+```javascript
+currency(123, { fromCents: true });               // 1.23
+currency('123', { fromCents: true });             // 1.23
+currency(123, { fromCents: true, precision: 0 }); // 0.123
+currency(123, { fromCents: true, precision: 3 }); // 123
```

Comment:
This doesn't seem right, shouldn't `currency(123, { fromCents: true, precision: 0 })` be `123`? I would also expect the inverse from `precision: 3` to be `0.123`

Reply:
Correct. Sorry about that.